cmd/bundle/run.go

```diff
@@ -19,7 +19,27 @@ func newRunCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "run [flags] KEY",
 		Short: "Run a resource (e.g. a job or a pipeline)",
-		Args:  root.MaximumNArgs(1),
+		Long: `Run a resource (e.g. a job or a pipeline)
+
+The KEY argument is the unique identifier of the resource to run. You can customize the run by
+specifying any of the flags below, or if applicable, additional positional arguments.
+
+Additional positional arguments can be specified like this:
```

Comment:
```suggestion
The KEY argument is the unique identifier of the resource to run. In addition to customizing the run using any of the available flags, you can also specify additional positional arguments like this:
```

Reply:
Modified the suggestion again because I think the first example is using keyword arguments?